apps/students/models.py

```diff
@@ -36,13 +39,16 @@ class Student(models.Model):
     remarks = models.TextField(max_length=5000, blank=True)
 
     def __str__(self):
-        return f'{self.get_full_name} ({self.school_class})'
+        return f'{self.get_full_name} ({self.school_class}) ({self.get_village}) ({self.get_profile_image_url})'
```

Comment:
`__str__` method is used to represent the student instance on the admin portal, you don't need to add village and profile_image URL here.

Reply:
removes extra fields..